ui/apps/platform/src/Containers/SystemHealth/CentralDatabaseHealth/CentralDatabaseHealthCard.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { ReactElement } from 'react';
+import {
+    Card,
+    CardBody,
+    CardHeader,
+    CardTitle,
+    Flex,
+    FlexItem,
+    Text,
+} from '@patternfly/react-core';
+
+import useRestQuery from 'hooks/useRestQuery';
+import { fetchDatabaseStatus } from 'services/DatabaseService';
+import { DatabaseStatus } from 'types/databaseService.proto';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+
+import { ErrorIcon, SpinnerIcon, SuccessIcon, WarningIcon } from '../CardHeaderIcons';
+
+function isDatabaseVersionSupported(databaseVersion: string | undefined): boolean {
+    if (!databaseVersion) {
+        return true;
+    }
+    return databaseVersion.startsWith('15');
+}
+
+function getDatabaseHealthInfo(data: DatabaseStatus | undefined): {
+    status: 'healthy' | 'unhealthy' | undefined;
+    messages: string[];
+} {
+    if (!data) {
+        return { status: undefined, messages: [] };
+    }
+
+    // If the database is externally managed (ACSCS) then we do not currently have error information to report
```

Comment:
This comment caught my eye. If the database is externally managed, do we not have access to error information (false positive), or is there no error information (true positive)?

Reply:
Good call out. ACSCS is going to remain on PG 13 for the near future, so we do not want to report errors in that environment when the mismatch is detected. Since the only case where an error should be shown is when Postgres is != 15 **and** the database is internally managed, there are no situations where this should show an error for ACSCS (currently).